dll/win32/comdlg32/lang/cdlg_Ro.rc

```diff
@@ -197,45 +197,45 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Imprimare"
 FONT 8, "MS Shell Dlg"
 {
- LTEXT "Imprimantă:", stc1, 6, 6, 40, 9
+ LTEXT "Imprimantă", stc1, 6, 6, 40, 9
```

Comment:
you stripped the `:` here. By intention, or was that an accident?

Reply:
Yes, I did it by intention. In XP/2K3 the : does not exist.

![2024-04-30 00_23_18-Resource Hacker - comdlg32 dll mui](https://github.com/reactos/reactos/assets/113553683/dee12fc7-4c6e-4adf-9b9f-749a8359e9e1)

Reply:
Resolved.